io.wavebox.Wavebox.yaml

```diff
@@ -0,0 +1,141 @@
+app-id: io.wavebox.Wavebox
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: wavebox
+separate-locales: false
+tags: [proprietary]
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --device=all
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland or not?

Reply:
Well, it's chromium based so I left it in there because it should work as chromium does however when I try it, it throws :

```
flatpak run io.wavebox.Wavebox --enable-features=UseOzonePlatform --ozone-platform=wayland
[2 preload-host-spawn-strategy] Warning: waitpid override ignores groups
[2 preload-host-spawn-strategy] Warning: waitpid override ignores groups
[2 preload-host-spawn-strategy] Warning: waitpid override ignores groups
[2 preload-host-spawn-strategy] Warning: waitpid override ignores groups
[2 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/wavebox/wavebox --type=zygote --enable-crashpad --crashpad-handler-pid=27 --enable-crash-reporter=,stable --change-stack-guard-on-fork=enable
[2:2:0123/203941.417127:ERROR:wayland_connection.cc(208)] Failed to connect to Wayland display
[2:2:0123/203941.417152:FATAL:ozone_platform_wayland.cc(231)] Failed to initialize Wayland platform
[0123/203941.417234:ERROR:scoped_ptrace_attach.cc(27)] ptrace: Operation not permitted (1)
```

You want that removed then ?

Reply:
Ok it's removed now.